.github/workflows/test-windows.yml

```diff
@@ -0,0 +1,16 @@
+name: e2e-kind
+on:
+  - pull_request
+
+jobs:
+  build-and-test:
+    runs-on: windows-latest
```

Comment:
assume the plan is to actually do tests here in windows?  Otherwise, we wouldn't need a windows box to build.

Reply:
Yes. This PR is still WIP

Reply:
Yeah I saw the WIP and draft labels 🤣, was just confirming the direction you were heading.